test/e2e/must-gather/must_gather_test.go

```diff
@@ -104,8 +107,13 @@ var _ = ginkgo.Describe("[must-gather] NRO data collected", func() {
 				err = checkfilesExist(crdInstances, mgContentFolder)
 				gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
-				ginkgo.By("Checking 'numaresources' namespace")
-				namespaceFolder := filepath.Join(mgContentFolder, "namespaces/numaresources")
+				ginkgo.By("Looking for namespace in NUMAResourcesOperator")
+				updatedNRO, err := wait.ForDaemonsetInNUMAResourcesOperatorStatus(e2eclient.Client, deployment.NroObj, 5*time.Second, 2*time.Minute)
```

Comment:
when the NRO status is Available, we should have daemonsets populated (bar unavoidable race). But we can improve later.